src/test/kotlin/com/dansplugins/fiefs/command/TestFiefsCommand.kt

```diff
@@ -0,0 +1,60 @@
+package com.dansplugins.fiefs.command
+
+import com.dansplugins.fiefs.Fiefs
+import com.dansplugins.fiefs.command.fiefs.FiefsCommand
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import org.bukkit.command.Command
+import org.bukkit.command.CommandSender
+import org.junit.jupiter.api.Test
+
+class TestFiefsCommand {
+
+    private fun createMockPlugin(): Fiefs {
+        val mockPlugin = mockk<Fiefs>() {
```

Comment:
You can probably pull this up to a field, JUnit creates a new instance of the test class on each run.

Reply:
Good call, this is now a field.